### DISCUSSION TITLE
Total Contributions Not Showing Private Contributions

### DISCUSSION THREAD

**Discussion by NikiRich** [DenverCoder1/github-readme-streak-stats#529] [Q&A]
Say this is the current code: 
src="https://streak-stats.demolab.com?user=username&theme=soft-green&border_radius=5&card_width=500" 

How can I make it to incorporate private repositories contributions? Using Vercel: [github-readme-streak-stats-username.vercel.app](https://github-readme-streak-stats-username.vercel.app/)

**Comment by DenverCoder1** [comment]
https://github.com/DenverCoder1/github-readme-streak-stats/blob/main/docs/faq.md#how-do-i-enable-private-contributions

![image](https://github.com/DenverCoder1/github-readme-streak-stats/assets/20955511/12d66a33-d27a-4794-9805-2b6bd4ab6874)

Do you have this setting enabled on your profile?

**Comment by NikiRich** [comment]
Yes, I do

**Reply by NikiRich** [reply]
Like I have 50 commits in the stats (which is again less) and I only have 10 contributions in the streak stats.

**Comment by DenverCoder1** [comment]
When I look at your stats, it shows 63 which is the same as what it shows next to your contribution graph.

![](https://cdn.discordapp.com/attachments/815674728114814977/1116692196687626260/image0.jpg)

The streak stats gets the info directly from what GitHub returns in the API.

**Reply by NikiRich** [reply]
Yeap, I see it got updated now, used to show only 10. Thanks!
